src/tests/ftest/util/ior_test_base.py

```diff
@@ -238,7 +238,12 @@ def run_ior(self, manager, processes, intercept=None, display_space=True,
         """
         env = self.ior_cmd.get_default_env(str(manager), self.client_log)
         if intercept:
-            env["LD_PRELOAD"] = intercept
+            env['LD_PRELOAD'] = intercept
+            env['D_LOG_MASK'] = 'INFO'
+            env['D_IL_REPORT'] = '1'
+            #env['D_LOG_MASK'] = 'INFO,IL=DEBUG'
+            #env['DD_MASK'] = 'all'
+            #env['DD_SUBSYS'] = 'all'
```

Comment:
Are these lines (DD_MASK and DD_SUBSYS) meant to be left here?

Reply:
Yes, it took me ages to find out how to enable debug in this test.